Code/src/main/java/nl/utwente/group10/ui/components/OutputBlock.java

```diff
@@ -0,0 +1,67 @@
+package nl.utwente.group10.ui.components;
+
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Label;
+import nl.utwente.ewi.caes.tactilefx.fxml.TactileBuilderFactory;
+
+import java.io.IOException;
+
+/**
+ * OutputBlock is an extension of Block that only provides a display of the input
+ * it receives through it's inputAnchor.
+ * The input will be rendered visually on the Block.
+ * OutputBlock can be empty and contain no value at all, the value can be altered at any time
+ * by providing a different input source using a Connection.
+ */
+public class OutputBlock extends Block {
+	
+	/** The input this Block is receiving.**/
+	private String input;
+	/** Determines if this Block serves output.**/
+	private boolean output;
+	
+	/**
+	 * Creates a new instance of OutputBlock should only ever be called
+	 * internally by the class, for instantiation see other newInstance method.
+	 * @return new OutputBlock instance
+	 * @throws IOException
+	 */
+    private static OutputBlock newInstance() throws IOException {
+        OutputBlock block = FXMLLoader.load(OutputBlock.class.getResource("/ui/OutputBlock.fxml"), null, new TactileBuilderFactory());
+        ((Label) block.lookup("#label_value")).setText("Connect Input");
+        return block;
+    }
+    
+    /**
+     * Creates a new OutputBlock instance, servesThroughput is a boolean value to determine
+     * if the newly created instance should initially put input through as output in addition to
+     * displaying the value.
```

Comment:
Design decision, naturally an OutputBlock with throughput enabled does indeed have the same functionality. Disabling it reduces visual children. In hindsight I have no other justification for this, class can indeed be simplified.